packages/apps/spaces/app/organizations/page.tsx

```diff
@@ -117,7 +220,24 @@ export default function OrganizationsPage() {
       (data?.pages?.flatMap(
         (o) => o.dashboardView_Organizations?.content,
       ) as Organization[]) || [],
-    [data],
+    [
+      data,
```

Comment:
Those are pretty long, it could be encapsulated in variable and spread here for better readability